examples/tool-response-injection/demo.py

```
@@ -0,0 +1,275 @@
+        )
+
+
+def run_tamper_test(result: dict[str, Any], pub_hex: str) -> tuple[bool, str]:
+    candidate = block_receipts(result["verified"])[0]
+    tampered = tamper_signature(candidate)
+    try:
+        verify_receipt_inline(tampered, pub_hex)
+    except InvalidSignature as exc:
+        detail = str(exc) or exc.__class__.__name__
+        return True, detail
+    except Exception as exc:  # pragma: no cover - unexpected failure still useful here
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)